src/taskpane/pages/dashboard.tsx

```diff
@@ -1,45 +1,27 @@
-import {
-  IStackStyles,
-  DefaultPalette,
-  ISearchBoxStyles,
-  Stack,
-  SearchBox,
-  PrimaryButton,
-  ButtonType,
-} from "@fluentui/react";
+import { IStackStyles, DefaultPalette, Stack } from "@fluentui/react";
 import React from "react";
-import Header from "../components/Header";
-/* global Word */
+import data from "../../utils/data";
+import { ReferenceList } from "../components/ReferenceList";
+import SearchField from "../components/SearchField";
+// /* global Word */
 
 const stackStyles: IStackStyles = {
   root: {
     background: DefaultPalette.white,
+    overflow: "auto",
   },
 };
-const searchBoxStyles: Partial<ISearchBoxStyles> = { root: {} };
 
 function Dashboard() {
-  const click = async () => {
-    return Word.run(async (context) => {
-      // insert a paragraph at the end of the document.
-      const paragraph = context.document.body.insertParagraph("Hello World!", Word.InsertLocation.end);
-      paragraph.font.color = "black";
-      paragraph.font.size = 30;
-      await context.sync();
-    });
+  const originalItems = data;
+  const [items, setItems] = React.useState(originalItems);
+  const onFilterChange = (_: any, text: string): void => {
+    setItems(originalItems.filter((item) => item.title.toLowerCase().indexOf(text.toLowerCase()) >= 0));
```

Comment:
My suggestion were to add a few extra checks to this part of the code, say if you want to add sorting on the year field,

```suggestion
    setItems(originalItems.filter((item) => item.title.toLowerCase().indexOf(text.toLowerCase()) >= 0) || item.year.indexOf(text) >= 0);
```

at least until we can get the required features in JabRef online. It probably will make sense to create a function that takes an `item` as an argument and change 

```suggestion
    setItems(originalItems.filter(containsSearchTerm);
```

to make it more readable. Does it make sense to search in title, keywords, year, and authors for now?